diffs/preAaveV3_Ethereum_GHOBorrowRateUpdate_20230904_postAaveV3_Ethereum_GHOBorrowRateUpdate_20230904.md

```diff
@@ -0,0 +1,45 @@
+## Reserve changes
+
+### Reserves altered
+
+#### GHO ([0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f](https://etherscan.io/address/0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x16E77D8a7192b65fEd49B3374417885Ff4421A74](https://etherscan.io/address/0x16E77D8a7192b65fEd49B3374417885Ff4421A74) | [0xA901Bf68Bebde17ba382e499C3e9EbAe649DF276](https://etherscan.io/address/0xA901Bf68Bebde17ba382e499C3e9EbAe649DF276) |
+| maxExcessUsageRatio | 0 % | 100 % |
+| baseVariableBorrowRate | 1.5 % | 2.5 % |
+| maxExcessStableToTotalDebtRatio | 0 % | 100 % |
```

Comment:
why this and `maxExcessUsageRatio` changes from 0 to 100%?

Reply:
dunno and unexpected, maybe GHO-specific?